### DISCUSSION TITLE
Bug? Odd drone/police behavior from Needlejets

### DISCUSSION THREAD

**Discussion by CuffDunk** [induktio/thinker#69] [Q&A]
Loving the mod! It's re-ignited my love of SMAC from years ago.

Noticing what I think is a bug where Needlejets I produce while in Free Market are triggering the "out of territory" drones regardless of whether they are in my territory or not. If this is desired behavior that I'm misremembering, apologies.

![Untitled](https://github.com/induktio/thinker/assets/170366457/6b492990-9009-4c24-8006-e76307c480cf)

**Comment by induktio** [comment]
It looks like just the vanilla game mechanic since aircraft units will always generate pacifism drones regardless of their location. Maybe it was added because they have the ability to easily move outside the territory borders and return on the same turn. Whether the feature should be modded is probably another question.

**Comment by CuffDunk** [comment]
Gotcha! Somehow I either never realized that was the mechanic or had
forgotten since the early days. Thanks!
